lib/app.js

```diff
@@ -32,16 +34,16 @@ module.exports = settings => {
 
   app.use(content);
 
-  app.param('versionId', (req, res, next, versionId) => {
-    req.versionId = versionId;
-    next();
-  });
-
   app.param('profileId', (req, res, next, param) => {
     req.profileId = param;
     next();
   });
 
+  app.param('versionId', (req, res, next, param) => {
+    req.versionId = param;
+    getVersion(req, res, next);
```

Comment:
This seems a bit odd here. Would make more sense encapsulated in the router to me. Is there a reason for pulling it out here?

Reply:
I put it here so it only gets called once for all /:versionId routes - if not it would need to go in the read and edit routers

Reply:
I think that's preferable to having it sitting here IMO.